src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-size: 12px;
+  font-family: Roboto, serif;
```

Comment:
Consider using a fallback font like 'sans-serif' instead of 'serif' for better consistency with the 'Roboto' font, which is a sans-serif font.